Sources/UAPusher/Builder/InApp.swift

```diff
@@ -0,0 +1,40 @@
+
+import JSON
+
+/// In-app segment of an Urban Airship push notification payload
+/// See: https://docs.urbanairship.com/api/ua/#api-in-app-object
+public final class InApp: BuildingBlock {
+    // MARK: Class fields
+    
+    /// InApp representation
+    var payload: JSON
+    
+    // MARK: Predefined values
+    
+    public enum Predefined: JSONRepresentable {
+        
+        case alert
```

Comment:
Same comment here as in `Campaigns`. This doesn't seem to add much value to the consumer.